### PR TITLE
Roman

### PR DESCRIPTION
**Pull request by RomanMachylo**

### CODE REVIEW

File: MainProject/Models/Transformator.cs [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+             { 0, 0, 0, 1 }};
+
+            var Shift = new float[4, 4]
+            {{ 1, 0, 0, ShiftX },
+             { 0, 1, 0, ShiftY },
+             { 0, 0, 1, ShiftZ },
+             { 0, 0, 0, 1 }};
+
+            TransformationMatrix = Multiply(Multiply(Shift, Rotate), Scale);
+        }
```
Comment (Hurmaze) [inline MainProject/Models/Transformator.cs:147]: Pizdez what is going on here

File: MainProject/Models/Transformator.cs [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+        //    ShiftY = 0;
+        //    ShiftZ = 0;
+        //    ScaleX = 1;
+        //    ScaleY = 1;
+        //    ScaleY = 1;
+        //    RotateX(-AngleRadX, false);
+        //    RotateY(-AngleRadY, false);
+        //    RotateZ(-AngleRadZ, true);
+        //}
+
```
Comment (Hurmaze) [inline MainProject/Models/Transformator.cs:161]: let's remove unused code

File: MainProject/Models/Transformator.cs [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+        public Point Apply(Point point)
```
Comment (Hurmaze) [inline MainProject/Models/Transformator.cs:162]: Call it `ApplyTransformation` or something like this to be clear

**Review by Hurmaze — APPROVED** [review]

File: MainProject/Models/Transformator.cs (hunk 1 above)
Comment (RomanMachylo) [reply MainProject/Models/Transformator.cs:147]: It is math bro, don't try to think about it))

File: MainProject/Models/Transformator.cs (hunk 2 above)
Comment (RomanMachylo) [reply MainProject/Models/Transformator.cs:161]: I forgot to delete before pushing, it is already done

**Comment by Hurmaze** [comment]
> _ No description provided. _

btw, in the future make more informative PR naming

File: MainProject/Models/Transformator.cs (hunk 1 above)
Comment (f1ctashka) [reply MainProject/Models/Transformator.cs:147]: жоскій пон

**Review by f1ctashka — APPROVED** [review]

**Pull request merged**